sunbird_service_api_test/src/test/java/org/sunbird/integration/test/otp/OtpVerificationTest.java

```diff
@@ -29,9 +31,17 @@
 
   public static final String TEST_VERIFY_OTP_FAILURE_WITH_INVALID_TYPE =
       "testVerifyOtpFailureWithInvalidType";
+  
+  public static final String TEST_VERIFY_OTP_FAILURE_WITH_INVALID_OTP =
+	      "testVerifyOtpFailreWithInvalidOtp";
+
+  public static final String TEST_GENERATE_OTP_SUCCESS_WITH_PHONE =
+	      "testGenerateOtpSuccessWithPhone2";
```

Comment:
testGenerateOtpSuccessWithPhone2 to testGenerateOtpSuccessWithPhoneForVerification